packages/palette/src/elements/Text/Text.tsx

```diff
@@ -0,0 +1,243 @@
+import { themeGet } from "@styled-system/theme-get"
+import {
+  color,
+  ColorProps,
+  compose,
+  ResponsiveValue,
+  style,
+  typography,
+  TypographyProps,
+  variant,
+} from "styled-system"
+import { styled as primitives } from "../../platform/primitives"
+import { Color } from "../../Theme"
+
+/**
+ * font-families
+ */
+export const TEXT_FONTS = {
+  sans: '"ll-unica77", "Helvetica Neue", Helvetica, Arial, sans-serif',
+  serif: '"adobe-garamond-pro", "Times New Roman", Times, serif',
+}
+
+/**
+ * font-size scale
+ */
+export const TEXT_FONT_SIZES = {
+  size12: "62px",
+  size11: "55px",
+  size10: "48px",
+  size9: "32px",
+  size8: "28px",
+  size7: "24px",
+  size6: "18px",
+  size5: "16px",
+  size4: "15px",
+  size3: "14px",
+  size2: "13px",
+  size1: "12px",
+}
```

Comment:
Explicitly named these with `sizeN` to avoid the scale index confusion and to line up with the naming convention design is using.